src/renderers/shared/utils/ReactErrorUtils.js

```diff
@@ -240,6 +244,17 @@ if (__DEV__) {
               'or switching to a modern browser. If you suspect that this is ' +
               'actually an issue with React, please file an issue.',
           );
+        } else if (isCrossOriginError) {
+          error = new Error(
+            "A cross-origin error was thrown. React doesn't have access to " +
+              'the actual error because it catches errors using a global ' +
+              'error handler, in order to preserve the "Pause on exceptions" ' +
+              'behavior of the DevTools. This is only an issue in DEV-mode; ' +
+              'in production, React uses a normal try-catch statement.\n\n' +
+              "It's recommended to serve JavaScript files from the same " +
+              'origin as your application.',
```

Comment:
I don't think so, @sebmarkbage. I created a [test page](https://github.com/unpkg/unpkg-demos/blob/master/basic-react-cross-origin-errors/index.html) that loads 16.0.0-beta.2 from unpkg w/out using `<script crossorigin>` and it seems to work ok (i.e. I can use "Pause on exceptions" w/out any problem) and React is able to catch and report the error in the console.

![screen shot 2017-08-02 at 5 10 45 pm](https://user-images.githubusercontent.com/92839/28900379-91c1f7c2-77a5-11e7-81bf-6f0823ff7466.png)

Is that what I'm supposed to be testing?